mnemosyne/cli.py

```
@@ -699,6 +699,264 @@ def _progress(store, done, total):
+                print(f"  {key}")
+        else:
+            print("Config unchanged.")
+
+    elif sub == "get":
+        if len(rest) < 1:
+            _fail("Key required: mnemosyne config get <key>")
+        key = rest[0]
+        config = get_config()
+        val = config.get(key)
+        if val is None:
+            print(f"(not set)")
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)